provision/minikube/loki.yaml

```diff
@@ -1,5 +1,15 @@
-#  https://github.com/grafana/helm-charts/tree/main/charts/loki
+#  https://github.com/grafana/helm-charts/blob/main/charts/loki/values.yaml
 
 serviceMonitor:
   enabled: true
 
+persistence:
+  enabled: true
+  accessModes:
+    - ReadWriteOnce
+  size: 1Gi
+
+config:
+  table_manager:
+    retention_deletes_enabled: true
+    retention_period: 60d
```

Comment:
could this retention period and the one from monitoring.yaml, be abstracted to values.yaml, so that we have a single place to control the value from

Reply:
This file is already a values file, and those values are IMHO best overwritten via Helm's CLI. 

I've now leveraged the "dotenv"-feature of task; you can now use `KB_RETENTION` in and `.env` file to override that value.